README.md

```diff
@@ -222,6 +222,10 @@ dip run bash -c pwd
 
 returned is `/app/sub-project-dir`.
 
+#### $DIP_CURRENT_USER
+
+Exposes the current user ID (UID). It is useful when you need to run a container with the same user as the host machine.
```

Comment:
Could you add some useful example?

Reply:
@bibendi I added our use case that we are using it for, but let me know if it seems a little wordy and I'll attempt to come up with a more concise example.